src/hello_client.erl

```diff
@@ -165,12 +166,16 @@ handle_call(terminate, _From, State) ->
 %% @hidden
 handle_info({?INCOMING_MSG, Message}, State) ->
     incoming_message(Message, State);
+handle_info(?PING, State = #client_state{waiting_for_pong = true, keep_alive_interval = KeepAliveInterval}) ->
+    ?LOG_ERROR("There is no PONG answer on PING for ~p msec. Client will be restarted.", [KeepAliveInterval]),
```

Comment:
"Client will be restarted"? Isn't it more the *connection* that will be *reestablished*?
*For ~p msec*, seems wrong too, because the argument is only the "KeepAliveInterval" and not elapsed time since last PING (now - last_ping). So when there was no PONG since 7 seconds, but the Interval is 5 seconds, it will incorrectly say "5 seconds", right?